I declare that the work presented in this thesis is my own unless otherwise stated, for example where my analyses used data from published work or data from collaborators. This thesis has not been submitted, either partially or in full, for another qualification of this University, or for a qualification at any other institution. 

\noindent
Much of the work in chapter 4 has been published in \cite{harman_kmt2a-aff1_2021}. I wrote the manuscript, analysed the data, and performed experiments in this publication and I declare that no work performed solely by others has been included, unless otherwise stated.

\noindent
\textbf{Collaborators and their contributions to this work:}

\noindent
For computational analyses presented in Chapters 3 and 4, I used experimental data generated by previous/current members of the de Bruijn and Milne labs, alongside published datasets. The sources of experimental data are indicated in the figure legends and in the main body of text. Unpublished data was sourced as follows:

\noindent
\textbf{Fig. \ref{fig:ch3_overview}} Unpublished mini-bulk RNA-seq and ATAC-seq data on cells undergoing EHT in the mouse embryo (de Bruijn group, L. Greder and G. Swiers) was used where I generated and explored an integrated network model of EHT.

\noindent
\textbf{Fig. \ref{fig:ch3_runx1-regulation}} Published tiled capture-C \citep{owens_dynamic_2022} was used to complement and extend the integrated EHT network model.

\noindent
\textbf{Fig. \ref{fig:ch3_runx1-ikzf1}} Unpublished Capture-C and qRT-PCR data (de Bruijn group, D. Owens and L. Greder) was used to complement findings from the EHT network.

\noindent
\textbf{Fig. \ref{fig:ch4_GRN_just}} Analysis performed by R. Thorne was included to introduce the hypothesis for chapter 4.

\noindent
\textbf{Fig. \ref{fig:ch4_peak-qc}} Unpublished MLL-N ChIP-seq data from patient samples (Milne group, N. Crump) was analysed and included to validate ChIP-seq peak calling.

\noindent
\textbf{Fig. \ref{fig:ch4_runx1-grn}} Unpublished nascent RNA-seq data following \textit{RUNX1} knockdown (Milne lab, M. Tapia) was used to generate a network model of RUNX1 in MLL-AF4 leukaemia.

\noindent
\textbf{Fig. \ref{fig:ch4_essentiality}} Analysis performed by T. Wilson on a published CRISPR essentiality screen (DepMap, Avana 21Q1, \cite{meyers_computational_2017, doench_optimized_2016}), under my supervision.

\noindent
\textbf{Fig. \ref{fig:ch4_multiome-clusters}, \ref{fig:ch4_multiome-erg}, \ref{fig:ch4_multiome-grn}, \ref{fig:ch4_multiome-tradeseq}} Single cell multiome data were generated in a collaboration between the Milne and Roy labs, involving myself, A. Smith, N. Elliott, and S. Rice. Data analysis was done by A. Smith and me: A. Smith contributed the pre-processing and quality control filtering of the ATAC-seq data. The rest of the analysis was performed by me.

\noindent
\textbf{Fig. \ref{fig:ch5_overview}} Mouse experiments (GMP isolation) were performed with assistance fom S. Valletta and S. Rice. The MLL-AF9 mCherry retroviral construct and \mybmre{} mouse models used in these experiments was designed and generated by I. Lau (Milne group).

